pareto/utilities/test_plot_bars.py

```diff
@@ -0,0 +1,29 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021 by the software owners: The
+# Regents of the University of California, through Lawrence Berkeley National Laboratory, et al. All
+# rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the
+# U.S. Government consequently retains certain rights. As such, the U.S. Government has been granted
+# for itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license
+# in the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit other to do so.
+#####################################################################################################
+from pareto.utilities.get_data import get_data
+from pareto.utilities.results import plot_bars
+from importlib import resources
+
+set_list = []
+parameter_list = ["test_plot_bar"]
+
+with resources.path("pareto.case_studies", "visualization_test_data.xlsx") as fpath:
+    [df_sets, df_parameters] = get_data(fpath, set_list, parameter_list)
+
+# Calling plot_bars using the get_data format
+args = {
+    "chart_title": "Test Data",
+    "labels": [("Origin", "Destination", "Time", "Value")],
+}
+input_data = {"pareto_var": df_parameters["test_plot_bar"]}
+plot_bars(input_data, args=args)
```

Comment:
I've added a suggestion on how to refactor this file so that it will be picked up by pytest:

- A test function (must be prefixed with `test_`), where the `plot_bars()` function will be called
- The input data, represented by the two fixtures `input_data` and `plot_args`, that are passed to the test function through the function's argument with the corresponding name
  - i.e. the `input_data` arg of `test_plot_bars()` will have the output of the `input_data()` fixture function when `test_plot_bars()` is run, and similarly for the `plot_args` arg and `plot_args()` fixture function
  - Fixtures (https://docs.pytest.org/en/latest/how-to/fixtures.html) are the preferred way to manage test data in pytest to separate the various phases of a test, i.e. setup (fixtures), assertion (test functions), and teardown (optional, also uses fixtures)

Reply:
@lbianchi-lbl Thanks a lot for this.